openmc/filter.py

```diff
@@ -109,7 +109,7 @@ def type(self, type):
         if type is None:
             self._type = type
         elif type not in FILTER_TYPES.values():
-            msg = 'Unable to set Filter type to "{0}" since it is not one ' \
+            msg = 'Unable to set Filter type to ""{0}" since it is not one ' \
```

Comment:
Fixed
On Aug 4, 2015 9:32 PM, "Paul Romano" <notifications@github.com> wrote:

> In openmc/filter.py
> <https://github.com/mit-crpg/openmc/pull/428#discussion_r36268654>:
>
> > @@ -109,7 +109,7 @@ def type(self, type):
> >          if type is None:
> >              self._type = type
> >          elif type not in FILTER_TYPES.values():
> > -            msg = 'Unable to set Filter type to "{0}" since it is not one ' \
> > +            msg = 'Unable to set Filter type to ""{0}" since it is not one ' \
>
> Mistake here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/428/files#r36268654>.
>